content/blog/2025-04-04-tmil-15/index.md

```diff
@@ -0,0 +1,145 @@
+- [xilem#899][]: Added documentation for lots of items and methods which were missing documentation.
+- [xilem#882][]: Adds the core of a virtual scrolling widget, to be used with medium or large scroll areas.
+
+ <!-- TODO: Screenshot of virtual scrolling? -->
+
+## Vello
+
+Vello is our GPU vector renderer.
+It can draw large 2D scenes with high performance, using GPU compute shaders for most of the work.
+
+- [vello#841][]: Fixed incorrect COLR Emoi rendering, which especially impacted users on Windows.
+  We backported this to create [Vello 0.4.1](https://github.com/linebender/vello/releases/tag/v0.4.1).
+- [vello#817][], [vello#877][]: Updates some of our GPU encoding code to not assume that allocators will give over-aligned allocations.
+
+We have also renamed the #gpu channel on Zulip to [#vello](https://xi.zulipchat.com/#narrow/channel/197075-vello), which better reflects the purpose of the channel.
+Progress on the sparse strips renderer has also been continuing at pace:
+
+- [vello#827][]: Added some core API types.
+- [vello#828][]: Imported the implementation of sparse strip production, without rasterisation (Vello Common).
+- [vello#830][], [vello#832][]: Added on CPU rasterisation of sparse strips, with integration tests (Vello CPU).
+- [vello#834][]: Added a tool (Vello Toy) for visualising different stages of the new pipeline.
+- [vello#831][]: Added the fragment shader based GPU rasteriser for sparse strips (Vello Hybrid).
+- [vello#872][]: Enable running Vello Hybrid using WebGL2.
+- [vello#867][]: Benchmarking harness for Vello Common and Vello CPU.
+
+<!-- TODO: Screenshot of WebGL example, in e.g. Firefox on Linux? -->
+
+The current state of the sparse strip renderers is that they can reliably render scenes where all elements are a solid colour.
+<!-- TODO: Mention new mini roadmap? -->
+There are some PRs where you can track ongoing work:
+
+- [vello#878][]: Add clipping to Vello CPU.
+- [vello#883][]: Implements text/glyph rendering.
+- [vello#879][]: Runs Vello Hybrid on Android. <!-- Is now the time to announce Android View? In what form? -->
+
+<!-- Maybe? Screenshot relevant for clipping? -->
+
+You can also follow the progress in [#vello > Vello Hybrid](https://xi.zulipchat.com/#narrow/channel/197075-vello/topic/Vello.20Hybrid) and other threads in [#vello](https://xi.zulipchat.com/#narrow/channel/197075-vello).
+We also are hosting weekly renderer office hours specifically aimed at developing this collaboration.
+Information about how to join these, follow [#office hours](https://xi.zulipchat.com/#narrow/channel/359642-office-hours).
+
+## Parley
+
+Parley is a text layout library.
+It handles text layout, mostly at the level of line breaking and resolving glyph positions.
+The biggest news in March was that an egui community member has opened [a PR][egui#5784] to replace egui's bespoke text handling with Parley, with buy-in from egui's maintainer.
```

Comment:
cc @valadaptive are you happy with this wording?

It might also be worth running it by Emil once it's ready.

Reply:
Hm, for some reason these pings never showed up in my GitHub notifications.

The wording sounds good to me!